evm-contracts/src/v0.6/dev/FluxAggregator.sol

```diff
@@ -828,4 +820,18 @@ contract FluxAggregator is AggregatorInterface, Owned {
     _;
   }
 
+  function add(uint256 a, uint256 b, uint256 bits) internal pure returns (uint256) {
```

Comment:
Should we break off the `add` and `sub` functions to their own mini-library? It shouldn't add to the bytecode since the compiler will just flatten everything before-hand anyway (MinSafeMath?).

Reply:
Just tried that on a branch I pushed up called, feature/#172323520-5.

Looks like Solidity is not as forgiving about parameter types when you use libraries, because I got this error:
```
error FluxAggregator.sol:188:36: TypeError: Member "sub" not found or not visible after argument-dependent lookup in uint32.
    address tail = oracleAddresses[oracleCount().sub(1)];
```